src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_LENGTH_OF_SEGMENT = 20;
+    private static final int FIGURE_COUNT = 5;
+
+    public Figure getRandomFigure() {
+        final Random random = new Random();
+        final int figureNumber = random.nextInt(FIGURE_COUNT);
+        final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Please, make `final Random random = new Random();` and `final ColorSupplier colorSupplier = new ColorSupplier();` class level fields
Don't forget to add `private`